ansible/pmm2/post-build-actions.yml

```diff
@@ -67,3 +68,11 @@
         group: pmm
         mode: '0775'
 
+    - name: Create grafana.log file    | Create 'srv/logs/grafana.log' file
```

Comment:
Not sure about this one. I would say that everything that can log to stdout/stderr should do so, and let supervisord capture those streams and rotate them. We want to move away from processes logging to files themselves, and from logrotate (and crond).